back/views/default.py

```diff
@@ -139,14 +139,19 @@ def remove_marker(request):
 
 @view_config(route_name='list_markers', renderer='json')
 def list_markers(request):
+    #min_date = ...
```

Comment:
Just obtain it from GET parameters.

I think it's something like `request.params.get('min_date')`

Reply:
You can test it by going to the "/list-markers?min_date=2020-03-03"

Reply:
Thanks, I did that.